elements/pf-dropdown/pf-dropdown-item.css

```diff
@@ -0,0 +1,91 @@
+:host {
+  --pf-c-dropdown__menu-item--Color: var(--pf-global--Color--dark-100, #151515);
```

Comment:
instead of setting these on `:host`, which would override any user settings, please make them fallbacks.